src/pages/debrief/debrief.selector.ts

```diff
@@ -94,13 +113,18 @@ export const getDangerousFaults = (faults: DangerousFaults): SeriousFaultsContai
  * Returns a container of array holding the propertyName and fullCompetencyLabel for each serious
  * fault recorded against a candidate.
  */
-export const getSeriousFaults = (faults: SeriousFaults): SeriousFaultsContainer[] => {
-  const faultsEncountered: SeriousFaultsContainer[] = [];
+export const getSeriousFaults = (faults: SeriousFaults): CommentedCompetency[] => {
```

Comment:
Feel like the function documentation should be updated as well.

Reply:
Fixed.